pandas/tests/frame/methods/test_drop_duplicates.py

```diff
@@ -471,3 +471,14 @@ def test_drop_duplicates_non_boolean_ignore_index(arg):
     msg = '^For argument "ignore_index" expected type bool, received type .*.$'
     with pytest.raises(ValueError, match=msg):
         df.drop_duplicates(ignore_index=arg)
+
+
+def test_drop_duplicates_pos_args_deprecation():
+    # test deprecation warning message for positional arguments GH#41485
+    df = DataFrame({"a": [1, 1, 2], "b": [1, 1, 3], "c": [1, 1, 3]})
+    msg = (
+        r"Starting with Pandas version 2\.0 all arguments of drop_duplicates except for "
```

Comment:
This line is likely too long, did you run the linting checks before submitting? If you enable pre-commit (`pre-commit install`) they'll be run for you, else you can run them manually with `pre-commit run --files <any file you've modified>`

Reply:
You're right, it was too long. I changed it in my latest commit. I was using git outside of my development environment for convenience. In my latest commits, I made sure to have pre-commit enabled, thanks for the hint.